src/System.Management.Automation/engine/runtime/Binding/Binders.cs

```diff
@@ -284,7 +284,7 @@ internal static BindingRestrictions CombineRestrictions(this DynamicMetaObject t
             var targetTemp = Expression.Variable(target.Expression.Type);
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-linux`
TabCompletion.Miscellaneous completion tests.Test history completion
```powershell
Expected strings to be the same, but they were different.
Expected length: 23
Actual length:   478
Strings differ at index 0.
Expected: 'Test history completion'
But was:  '$env:POWER...'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/TabCompletion/TabCompletion.Tests.ps1: line 909
909:             $res.CompletionMatches[0].CompletionText | Should -BeExactly "Test history completion"
```